spec/take_away_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'take_away'
+
+describe TakeAway do
+
+  let(:quantity) { 6 }
+  let(:dish) { 'Scrambled eggs' }
+
+  default_menu = {
+    'Nutella on rice cakes and banana on top' => 3.00,
+    'Scrambled eggs' => 4.99,
+    'Roasted aubergine & tomato curry' => 5.60,
+    'Jacket Potato with whipped feta' => 8.25,
+    'Mushroom, ricotta & rocket tart' => 9.20,
+    'Cod with beans & artichokes' => 11.40,
+  }
+
+  describe "#menu output" do
+    it 'should output a list with dishes and prices' do
+      expect(subject.list).to eq default_menu
```

Comment:
This test doesn't feel quite correct to me - looking at the method in the Takeaway class, it seems like it should be printing out a formatted version on the menu, whereas this test seems to be checking that it returns a hash.

You might want to check up on how to use rspec test for something being printed to the console.